### DISCUSSION TITLE
How to change color limits in plot

### DISCUSSION THREAD

**Discussion by John-Luick** [OpenDrift/opendrift#887] [General]
Very simple question (once you know the answer?).
How to change the limits on the colors (and colorbar)?
I tried different things, for example "clim":
om.plot(buffer=0.1, show_particles='True', clim=(-0.4,0), cmap='rainbow', linecolor='z')
...to change the range to [-0.4, 0], but it had no effect. My python is very rudimentary (I normally do all my plotting outside opendrift in Matlab), but this would be helpful for a specific purpose.
Thanks
John

**Comment by knutfrode** [comment]
Hi, the correct for limits is `vmin` and `vmax`, see e.g. this example:
https://opendrift.github.io/gallery/example_radionuclides.html#sphx-glr-gallery-example-radionuclides-py

**Reply by John-Luick** [reply]
Thanks! That worked perfectly. I had looked through a number of gallery examples, and even did a search on some keyword, but somehow missed the radionuclide one.
